extension/src/lambda.js

```diff
@@ -9,8 +9,8 @@ exports.handler = async function (event) {
 
     return {
       responseType: paymentResult.success ? 'UpdateRequest' : 'FailedValidation',
-      errors: paymentResult.data.errors,
-      actions: paymentResult.data.actions
+      errors: paymentResult.data ? paymentResult.data.errors : undefined,
+      actions: paymentResult.data ? paymentResult.data.actions : undefined
```

Comment:
Could you please add a comment to the code from your PR description as it is not obvious to the reader:
`Null check around paymentResult.data, which can be null if paymentHandler short circuits when not an adyen payment`